lib/LaTeXML/Package/pstricks.sty.ltxml

```diff
@@ -821,7 +843,10 @@ DefSimplePSConstructor('\psgrid', 'PSCoord PSCoord PSCoord',
         LookupValue('\psyunit')->multiply(10)); }
     elsif (!$p1 && $p0) { $p2 = $p0; $p0 = ZeroPair; $p1 = $p0; }
     elsif (!$p2 && $p1) { $p2 = $p1; $p1 = $p0; }
-    $_[1]->setProperties(p0 => $p0, p1 => $p1, p2 => $p2); });
+    $_[1]->setProperties(p0 => $p0, p1 => $p1, p2 => $p2,
+      x0 => $p0->getX->pxValue, y0 => $p0->getX->pxValue,
+      x1 => $p1->getX->pxValue, y1 => $p1->getX->pxValue,
+      x2 => $p2->getX->pxValue, y2 => $p2->getX->pxValue); });
```

Comment:
shouldn't the `y0`, `y1` and `y2` here call `getY` ?

Reply:
Good catch